<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserManagerDialog</class>
 <widget class="QDialog" name="UserManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户权限管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>用户权限管理系统</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBoxUserList">
       <property name="title">
        <string>用户列表</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="tableUsers">
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btnRefresh">
            <property name="text">
             <string>刷新列表</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAddUser">
            <property name="text">
             <string>添加用户</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDeleteUser">
            <property name="text">
             <string>删除用户</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxPermission">
       <property name="title">
        <string>权限设置</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelUsername">
            <property name="text">
             <string>用户名：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="editUsername">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelGroup">
            <property name="text">
             <string>权限组：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboGroup">
            <item>
             <property name="text">
              <string>无权限组</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>权限组1 (分系统1-3)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>权限组2 (分系统4-5)</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="labelRestrictions">
          <property name="text">
           <string>个人权限限制（勾选表示禁止访问）：</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkSubsystem1">
            <property name="text">
             <string>禁止访问分系统1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="checkSubsystem2">
            <property name="text">
             <string>禁止访问分系统2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkSubsystem3">
            <property name="text">
             <string>禁止访问分系统3</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="checkSubsystem4">
            <property name="text">
             <string>禁止访问分系统4</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkSubsystem5">
            <property name="text">
             <string>禁止访问分系统5</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="labelPreview">
          <property name="text">
           <string>最终权限预览：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textPreview">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="maximumHeight">
           <number>80</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="btnSave">
            <property name="text">
             <string>保存权限设置</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnReset">
            <property name="text">
             <string>重置</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStatusBar" name="statusBar"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
